README.md

```diff
@@ -47,26 +51,83 @@ make deploy
 
 ## Running an ad-hoc shred
 
-The following steps will perform the shredding on the account-id specified earlier. ( the `ACCOUNT_ID_TO_SHRED` environment variable ).
-
-> DOUBLE CHECK THAT YOU'VE CONFIGURED THE CORRECT ACCOUNT ID, AS THIS IS A DESTRUCTIVE OPERATION THAT CAN NOT BE UNDONE!
-
-Run the following to create the account:
+Generally speaking, you should first try to shred an account by finding the official Account CR on the appropriate hive cluster and setting its state to "Failed"
+(e.g. `osdctl account set somehow-identified-account-cr-name --state=Failed`).
+
+If you cant do this for some reason, you can deploy the AWS Account Shredder locally, create an Account CR for the AWS Account IDs, marke them failed and let your
+local shredder clean them up. Use cases for this are predominately around cleaning up orphaned accounts from developer activity in staging/integration envrionments 
+(rather than customer accounts in production which should have a more sane state). In other words, this method should only be used as a last resort for AWS resources
+with no associated Account CR or hive cluster.
+
+The following steps will perform the shredding on the AWS account-ids specified earlier. ( the `AWS_ACCOUNTS_TO_SHRED_FILE` environment variable ).
+
+> DOUBLE CHECK THAT YOU'VE ACCOUNT IDS IN FILE, AS THIS IS A DESTRUCTIVE OPERATION THAT CAN NOT BE UNDONE!
+
+```
+make shred-accounts
+```
+
+Shredding can take minutes per account, so a large list of AWS Account IDs can take a long time to finish. 
+You can check on the status of the shredding with:
+```
+make shred-accounts-status
+```
+
+You can remove successfully shredded accounts with:
+```
+make shred-accounts-clean
+```
+
+Example:
+```
+$ oc status
+In project default on server https://api.crc.testing:6443
+
+svc/openshift - kubernetes.default.svc.cluster.local
+svc/kubernetes - 10.217.4.1:443 -> 6443
+
+View details with 'oc describe <resource>/<name>' or list resources with 'oc get all'.
+$ oc whoami
+kubeadmin
+$ cat ~/aws_account_ids.txt
+1234
+9876
+$ export AWS_ACCOUNTS_TO_SHRED_FILE=~/aws_account_ids.txt
+$ make predeploy
+...
+$ make deploy
+...
+$ make shred-accounts
+hack/get_current_api_url.sh | grep '127.0.0.1\|api.crc.testing'
+https://api.crc.testing:6443
+hack/shred_accounts.sh -f /Users/mstratto/aws_account_ids.txt mark
+Marking accounts for shredding
+account.aws.managed.openshift.io/aws-shredder-account-delete-1234 created
+account.aws.managed.openshift.io/aws-shredder-account-delete-9876 created
+$ make shred-accounts-status
+hack/get_current_api_url.sh | grep '127.0.0.1\|api.crc.testing'
+https://api.crc.testing:6443
+hack/shred_accounts.sh -f /Users/mstratto/aws_account_ids.txt status
+Checking account shredder status.
+1234 - pending
+9876 - Ready
+Accounts: 2
+Accounts shredded: 1
+Account pending: 1
+Accounts missing: 0
+$ make shred-accounts-cleanup
```

Comment:
Can you add some spacing in between some of the commands to make it easier to digest?